f3/granite.go

```diff
@@ -424,25 +440,26 @@ func (i *instance) tryPrepare() {
 func (i *instance) beginCommit() {
 	i.phase = COMMIT
 	i.phaseTimeout = i.alarmAfterSynchrony(PREPARE)
-	i.broadcast(COMMIT, i.value, nil)
+	i.broadcast(i.round, COMMIT, i.value, nil)
 }
 
 func (i *instance) tryCommit(round uint32) {
 	// Unlike all other phases, the COMMIT phase stays open to new messages even after an initial quorum is reached,
 	// and the algorithm moves on to the next round.
-	// A subsequent COMMIT message can cause the node to decide, so there is no check on the current phase.
+	// A subsequent COMMIT message can cause the node to terminate, so there is no check on the current phase.
```

Comment:
```suggestion
	// A subsequent COMMIT message can cause the node to decide, so there is no check on the current phase.
```

A bit too happy with the find-and-replace. Do you use an IDE for such renames?

Reply:
Using GoLand these days, though I have a background as an emacs lover.